one-app-statics/package.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
I am not sure if we should commit the `package.json` file, I haven't added the script to the build scripts and I was thinking only running it when deploying the app... We could find ourselves updating this file after manually running the build or committing for something else. Thoughts?

Reply:
maybe generate at buildtime and include in .gitignore ?